server/server.go

```diff
@@ -510,17 +510,26 @@ func (s *Server) startMonitoring(secure bool) error {
 		hp           string
 		err          error
 		httpListener net.Listener
+		port   int
```

Comment:
And that begs the question: how is it going to make it better? If we have so many conflicts right now with 4222.8222 or any of the ports we use in other packages (there are not that many). My point is that it seems that we are not lucky that we got so many conflicts. That's why I said that I wonder if there is not something (else) going on.